empress/support_files/js/empress.js

```diff
@@ -742,6 +742,22 @@ define([
         return new Float32Array(coords);
     };
 
+    /**
+     * Returns the number of lines/trianges to approximate an arc/wedge given
+     * the total angle of the arc/wedge.
+     *
+     * @param {Number} totalAngle The total angle of the arc/wedge
+     * @return {Number} The number of lines/triangles to approximate the arc
+     *                  or wedge.
+     */
+    Empress.prototype._numSampToApproximate = function(totalAngle) {
+        var numSamples = Math.floor(
+            60 * Math.abs(totalAngle / Math.PI)
+        );
+        numSamples = numSamples >= 2 ? numSamples : 2;
+        return numSamples;
```

Comment:
```suggestion
        return numSamples >= 2 ? numSamples : 2;
```
(this is subjective, feel free to ignore it)